src/strategies/jwt.bearer.strategy.js

```diff
@@ -103,6 +103,27 @@ const verify = (jwt_payload, done) => {
         }
         if (isUser) {
             context['isUser'] = isUser;
+            // Test that required fields are populated
+            let validInput = true;
+            if (!jwt_payload['custom:clientFhirPersonId']) {
+                logDebug('Error: custom:clientFhirPersonId field is missing', {user: ''});
+                validInput = false;
+            }
+            if (!jwt_payload['custom:clientFhirPatientId']) {
+                logDebug('Error: custom:clientFhirPersonId field is missing', {user: ''});
+                validInput = false;
+            }
+            if (!jwt_payload['custom:bwellFhirPersonId']) {
+                logDebug('Error: custom:bwellFhirPersonId field is missing', {user: ''});
+                validInput = false;
+            }
+            if (!jwt_payload['custom:bwellFhirPatientId']) {
+                logDebug('Error: custom:bwellFhirPatientId field is missing', {user: ''});
+                validInput = false;
+            }
```

Comment:
I think we can reduce the duplicated code by creating an array of required payload attributes and logging whichever is not present in a loop. That would make it extensible as well. Thoughts?

Reply:
Good idea, implemented